tests/modules/site-health/audit-full-page-cache/audit-full-page-cache-test.php

```diff
@@ -0,0 +1,367 @@
+<?php
+/**
+ * Tests for audit-full-page-cache module.
+ *
+ * @package performance-lab
+ */
+
+class Audit_Full_Page_Cache_Tests extends WP_UnitTestCase {
+
+	const REST_NAMESPACE = 'performance-lab/v1';
+	const REST_ROTE      = '/tests/page-cache';
+
+	/**
+	 * Tests perflab_afpc_add_full_page_cache_test().
+	 */
+	public function test_perflab_afpc_add_full_page_cache_test() {
+		$result = perflab_afpc_add_full_page_cache_test( array() );
+		$this->assertEqualSets(
+			array(
+				'async' => array(
+					'perflab_page_cache' => array(
+						'label'             => __( 'Page caching', 'performance-lab' ),
+						'test'              => rest_url( 'performance-lab/v1/tests/page-cache' ),
+						'has_rest'          => true,
+						'async_direct_test' => 'perflab_afpc_page_cache_test',
+					),
+				),
+			),
+			$result
+		);
+	}
+
+	/**
+	 * Tests all possible scenarios given by dataProvider.
+	 *
+	 * @dataProvider provider_perflab_afpc_page_cache_test
+	 * @covers ::perflab_afpc_page_cache_test()
+	 * @covers ::perflab_afpc_get_page_cache_detail()
+	 * @covers ::perflab_afpc_page_cache_detected_test()
```

Comment:
See above, can you remove this line based on my other comment to remove this function?